pkg/policy/api/icmp.go

```diff
@@ -0,0 +1,72 @@
+package api
```

Comment:
Nit: We should add the copyright headers. Feel free to copy-paste from an existing file, but just be sure to update the year

Reply:
I see. Thank you :)